src/ctree-app/ctree-app.html

```diff
@@ -411,9 +432,19 @@
           if (!pageData.icon.startsWith(this.rootPath)) {
             pageData.icon = this.rootPath + pageData.icon;
           }
-          newNavList.push({name: name, data: pageData});
+
+          if(this.searchText){
+            if(pageData.label.toLowerCase().indexOf(this.searchText.toLowerCase()) !== -1){
+                newNavList.push({name: name, data: pageData});
+            }
+          }else{
+            if(pageData.saved){
```

Comment:
Please follow the code style conventions of the rest of the code and put spaces between keywords like `if` and `else`, and operators like `(` and `{`.  For example `if (this.searchText) {` and `} else {`.